src/Http/Http.Extensions/gen/StaticRouteHandlerModel/StaticRouteHandlerModel.Emitter.cs

```diff
@@ -177,11 +175,11 @@ private static void EmitBuiltinResponseTypeMetadata(this Endpoint endpoint, Code
 
         if (responseType.SpecialType == SpecialType.System_String)
         {
-            codeWriter.WriteLine("options.EndpointBuilder.Metadata.Add(new GeneratedProducesResponseTypeMetadata(type: null, statusCode: StatusCodes.Status200OK, contentTypes: GeneratedMetadataConstants.PlaintextContentType));");
+            codeWriter.WriteLine("options!.EndpointBuilder.Metadata.Add(new GeneratedProducesResponseTypeMetadata(type: null, statusCode: StatusCodes.Status200OK, contentTypes: GeneratedMetadataConstants.PlaintextContentType));");
```

Comment:
Do we need the null suppression if the `Debug.Assert` has been placed into the generated code?

Reply:
Seems we do. I put the Debug.Assert in there as a way of trying to avoid the null suppression but it didn't work. I haven't figure out why yet, was going to do some experimentation. That said, Debug.Assert is just a really verbose null suppression operator ;)